src/veritysetup/veritysetup.c

```diff
@@ -81,6 +82,35 @@ static int run(int argc, char *argv[]) {
                         return 0;
                 }
 
+                if (argc > 7 && strcmp(argv[7], "-")) {
+                        for (;;) {
+                                _cleanup_free_ char *word = NULL;
+
+                                r = extract_first_word((const char **)&argv[7], &word, ",", EXTRACT_DONT_COALESCE_SEPARATORS | EXTRACT_UNESCAPE_SEPARATORS);
+                                if (r < 0)
+                                        return log_error_errno(r, "Failed to parse options: %m");
+                                if (r == 0)
+                                        break;
+
+                                if (strcmp(word, "ignore-corruption") == 0)
```

Comment:
please use `streq(…)` instead of `strcmp(…) == 0` in our codebase as a matter of coding style.